steven_torrence/Ruby_OOP/lion.rb

```diff
@@ -0,0 +1,32 @@
+require_relative "mammal"
+
+class Lion < Mammal
+    def initialize
+        @health = 170
+    end
+
+    def fly
+        @health -= 10
+        p "The lion is flying!"
+        self
+    end
+
+    def attack_town
+        @health -= 50
+        p "The lion is attacking the town!"
+        self
+    end
+
+    def eat_humans
+        @health += 20
+        p "The lion just ate a human."
+        self
+    end
+
+    def display_health
```

Comment:
I believe this method is just overriding the parent display_health method. I wonder if there is a way to call this method and have it use the parent display_health and also the lion's display_health method. hint: super

Reply:
Okay, I was confused about that. If I use super, I still need to have a method in the lion class that  prints the updated text, right?